ppml/services/kms-utils/docker/entrypoint.sh

```diff
@@ -43,7 +43,7 @@ elif [ "$action" = "generatekeys" ]; then
 		--simpleAPPKEY $appkey
 	elif [ "$KMS_TYPE" = "azure" ]; then
 	    keyVaultName=$2
-		java -cp $BIGDL_HOME/jars/bigdl-ppml-spark_3.1.2-2.1.0-SNAPSHOT.jar:$SPARK_HOME/jars/*:$SPARK_HOME/examples/jars/*:$BIGDL_HOME/jars/* \
+		java -cp $BIGDL_HOME/jars/bigdl-ppml-spark_3.1.2-2.1.0-SNAPSHOT.jar:$BIGDL_HOME/jars/*:$SPARK_HOME/examples/jars/*:$SPARK_HOME/jars/* \
```

Comment:
why change the order

Reply:
there is a conflict jar in $BIGDL_HOME/jars/* and $SPARK_HOME/jars/*, put $SPARK_HOME/jars/* before $BIGDL_HOME/jars/* will cause NoSuchMethodError.